Mathlib/Algebra/Polynomial/Coeff.lean

```diff
@@ -140,7 +140,6 @@ theorem coeff_X_mul_zero (p : R[X]) : coeff (X * p) 0 = 0 := by simp
 theorem coeff_C_mul_X_pow (x : R) (k n : ℕ) :
     coeff (C x * X ^ k : R[X]) n = if n = k then x else 0 := by
   rw [C_mul_X_pow_eq_monomial, coeff_monomial]
-  congr 1
```

Comment:
This could make a performance difference, in theory... let's make sure to benchmark this PR!

Reply:
Benchmarking suggests no substantial change.